test/test_nn.py

```diff
@@ -12342,6 +12343,114 @@ def local_backward_hook(m, input, output):
         output.backward(torch.ones(5, 5), retain_graph=True)
         self.assertTrue(local_backward_called and global_backward_called)
 
+
+class LazyModule(torch.nn.modules.lazy.LazyModuleMixin, torch.nn.Module):
+    pass
+
+
+class TestLazyModules(TestCase):
+
+    def test_lazy_module_parameter(self):
```

Comment:
It would be interesting to have something similar to this for Linear/LazyLinear.
In particular you can add a test to ensure that you can load a Linear into a LazyLinear.